src/rebound/rebound.ts

```diff
@@ -19,38 +19,53 @@ export class Rebound {
 
   /**
    * An object used to keep track of the proper window
-   * @property {Window} reciever
+   * @protected {Window} reciever
    * @private
    */
-  private _reciever: Window;
+  protected _reciever: Window;
 
   /**
    * A boolean that tells you if you are inside the iframe or not
-   * @property {Boolean} isChild
+   * @protected {Boolean} isChild
    * @private
    */
-  private _isChild: boolean;
+  protected _isChild: boolean;
 
   /**
    * A string that contains a random id value used to tell rebound that the
    * events are still from rebound
-   * @property {String} events
-   * @private
+   * @private {String} events
+   * @protected
    */
-  private _randId: string;
+  protected _randId: string;
+
+  protected _isHammerClient: boolean;
```

Comment:
this is an unused variable... I would suggest adding the "no-unused-variable" property to tslint

Reply:
👕 Should remove this one as `ClientType` provides a way to tell HammerSpace that it's not a HammerSpace instance on the other side.

Reply:
Will remove.